src/nl/surf/eduhub_rio_mapper/endpoints/api.clj

```diff
@@ -39,12 +39,16 @@
             [ring.middleware.json :refer [wrap-json-response]]
             [ring.util.response :as response])
   (:import [java.net MalformedURLException URL]
-           java.util.UUID))
+           java.util.UUID
+           [java.time LocalDateTime]
+           [java.time.format DateTimeFormatter]))
 
 (def server-stopping (atom false))
 
 (def nr-active-requests (atom 0))
 
+(def datetime-formatter (DateTimeFormatter/ofPattern "yyyy-MM-dd HH:mm:ss"))
```

Comment:
Voor het verwerken van logs is het denk ik beter om overal UTC te gebruiken ipv "local time". We weten niet welke timezone er op die server ingesteld is en met deze formatting kan je dat ook niet meer terugzien.

De simpelste betrouwbare methode om een ISO-8601 string van UTC timestamp te krijgen is

```
(str (java.time.Instant/now))
```

https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#toString--